### ISSUE TITLE
problem running python3 encode.py -i ../dataset/ -e encodings.pickle -d hog

### ISSUE THREAD

**Issue by koenkv** [nischi/MMM-Face-Reco-DNN#90]
hello,

if I try to run python3 encode.py -i ../dataset/ -e encodings.pickle -d hog I get the following error:

`
pi@raspberrypi:~/MagicMirror/modules/MMM-Face-Reco-DNN/tools $ python3 encode.py -i ../dataset/ -e encodings.pickle -d hog
Traceback (most recent call last):
  File "encode.py", line 5, in <module>
    from imutils import paths
  File "/home/pi/.local/lib/python3.7/site-packages/imutils/__init__.py", line 8, in <module>
    from .convenience import translate
  File "/home/pi/.local/lib/python3.7/site-packages/imutils/convenience.py", line 6, in <module>
    import cv2
  File "/usr/local/lib/python3.7/dist-packages/cv2/__init__.py", line 89, in <module>
    bootstrap()
  File "/usr/local/lib/python3.7/dist-packages/cv2/__init__.py", line 79, in bootstrap
    import cv2
ImportError: libtesseract.so.4: cannot open shared object file: No such file or directory
`
im running it on a raspberry pi 4
can anybody help me what I do wrong?

**Comment by Dennisd80** [comment]
https://github.com/nischi/MMM-Face-Reco-DNN/issues/59#issuecomment-701476179

**Comment by mikecheng2626** [comment]
I can open up MagicMirror as a black screen and the notification showed that it recognized me as mike.

This is my module in config.js for MMM-Face-Reco-DNN part:

[22.txt](https://github.com/nischi/MMM-Face-Reco-DNN/files/6565988/22.txt)

I then progressed to MMM-Face-Multi-User-Recognition-SMAI, followed installation instruction.
I checked the public folder and there is no "mike" image, only "guest" image :((((

The encoding portion is like this... 

![1](https://user-images.githubusercontent.com/63714824/120099501-544fa080-c16e-11eb-8b5a-f28200ace1c1.PNG)

The instruction said it will take a while from a macOS, but I was running python3 encode.py directly on rpi since I only have 4 images of myself. I'm not sure if serializing encoding... means it's good?

What should I do?
